### ISSUE TITLE
5.0 tests pass when they shouldn't

### ISSUE THREAD

**Issue by djay** [collective/collective.collectionfilter#79]
The current robot tests assumes the code is running in ajax mode. The code supports both ajax and non-ajax mode (except for plone 5.0 which can't support ajax loading due to various bugs in the ajax load pattern). 

The current robot tests pass on plone 5.0 which means something is wrong with them

```
   Input text  css=.collectionSearch input[name='SearchableText']  Document
    # Text input is reloaded live ... wait a bit
    Should be 1 collection results
```

Actually has 3 collection results

![selenium-screenshot-3](https://user-images.githubusercontent.com/41700/62684809-ad73a080-b9eb-11e9-81c1-2c8475863432.png)

The test needs to be fixed so it actually fails for this.
Then the tests changed to have a set of tests that run in ajax mode (not run for on 5.0) and a set of tests that run in non-ajax mode. Or the same set of tests that work in both ajax and non ajax mode but run with 2 different layers which alter ajax-mode.

**Comment by djay** [comment]
actually maybe the easiest way is to 
- move the ajax search code into a seperate .robot file.
- have two layers, one with ajax and one without
- change the test_robot.py file to choose which robot files we want to run and run both on the two different layers

**Comment by JeffersonBledsoe** [comment]
Fixed by #82 and #72
